.github/workflows/build.yaml

```diff
@@ -75,3 +75,33 @@ jobs:
           pip install -e .
           pytest -vv tests/
 
+  test-fedora:
+    runs-on: ubuntu-latest
+    container: 'fedora:37'
+    needs: lint
+    steps:
+      - id: dependencies
+        run: sudo dnf install -y git-core make
+
+      - id: checkout-code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+
+      # See: https://github.com/actions/runner-images/issues/6775#issuecomment-1377299658
+      - id: chown-container-working-directory
+        name: Change Owner of Container Working Directory
+        run: chown root:root .
+
+      - id: build-and-test
+        name: Build and test
+        run: |
+          make build-dep-fedora
+          make rpm
+
+      - id: install
+        name: Install
+        run: sudo dnf install -y rpms/noarch/*.rpm
+
+      # test step is not needed (tests run on a build)
```

Comment:
Hmm, on build we run `make build-dep-fedora rpm`. I don't think we run tests there

Reply:
`make rpm` executes pytest. We can change it, but it should be a separate PR.

Reply:
I don't think it does. Also, `make rpm` takes about 2 min on my machine. Really not enough to run the tests. :laughing:

Reply:
Also (https://github.com/aiven/aiven-client/actions/runs/4014851129/jobs/6895903606):
![image](https://user-images.githubusercontent.com/3449635/215509768-7d84f306-3662-4403-b815-ca755b501a1d.png)

Reply:
Yes, you're right. I was looking in a wrong place.